app/models/uphold_connection.rb

```diff
@@ -91,6 +91,15 @@ class UpholdAccountState
     with_active_connection.with_expired_tokens
   }
 
+  scope :payable, -> {
+    joins(:publisher)
+      .with_active_connection
```

Comment:
This puts all the scopes for all wallet providers using the same `payable` scope for consistency.

We also add `with_active_connection`, so that if we can't refresh your connection, you are not payable.